cypress/e2e/inventory/search/browse-contributors-exact.cy.js

```diff
@@ -51,6 +51,7 @@ describe('ui-inventory: search', () => {
   });
 
   it('C353639 Browse contributors with exact match query (spitfire)', { tags: [testType.smoke, devTeams.spitfire] }, () => {
```

Comment:
test is failed, please check it

Reply:
The test is passed https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/527/allure/